### PR TITLE
scripts: Fix CMake spelling

### PR DESCRIPTION
**Pull request by rettichschnidi**
As per its creators, CMake is written with a capital "M".

If approved I'll also fix the spelling in areas other than the scripts folder.

<del>Note: CI (compliance checks) is failing due to what seems unrelated findings? I'd rather not address that in this PR.</del> suppressed

### CODE REVIEW

File: scripts/tests/twister_blackbox/test_outfile.py
Comment (golowanow) [inline scripts/tests/twister_blackbox/test_outfile.py:1]: `# pylint: disable=no-name-in-module`
at line 17 to get rid of E0611 from compliance check there

**Review by golowanow — DISMISSED** [review]

**Review by hakehuang — DISMISSED** [review]

**Comment by rettichschnidi** [comment]
Any chance to get this merged before RC1 hits? :pray:

**Review by LukaszMrugala — DISMISSED** [review]

**Review by thedjnK — DISMISSED** [review]

**Review by nashif — DISMISSED** [review]

**Comment by nashif** [comment]
> Any chance to get this merged before RC1 hits?

that is a fix that can be merged after RC1 as well.

**Comment by henrikbrixandersen** [comment]
Please rebase on `main` and resolve any merge conflicts.

**Comment by henrikbrixandersen** [comment]
Please rebase on `main` and resolve any merge conflicts.

**Review by golowanow — APPROVED** [review]

**Review by henrikbrixandersen — APPROVED** [review]

**Review by dkalowsk — APPROVED** [review]

**Review by thedjnK — APPROVED** [review]

**Comment by mmahadevan108** [comment]
@nashif , can you please approve again.

**Review by kartben — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Pull request merged**